src/main/java/com/conveyal/gtfs/validator/ReversedTripsValidator.java

```diff
@@ -36,21 +39,20 @@ public boolean validate(GTFSFeed feed, boolean repair, Double distanceMultiplier
     @Override
     public boolean validate(GTFSFeed feed, boolean repair) {
         boolean isValid = true;
-        ValidationResult result = new ValidationResult();
         int errorLimit = 5000;
```

Comment:
I added `errorLimit` after a discussion with @abyrd and you I think.  The idea was that if we're catching up to some large `n` of errors, then additional errors may not be valuable to have because there is probably something devastatingly wrong with the feed and it will rather just slow down loading of the feed (and make the validation report excessively large and burdensome to download).

We can change this if we think it's not actually a burden to churn through all of the errors, but just wanted to get the original purpose out there.